nltk/tag/senna.py

```diff
@@ -77,29 +79,41 @@ class SennaTagger(TaggerI):
     def __init__(self, senna_path, operations, encoding='utf-8'):
         self._encoding = encoding
         self._path = path.normpath(senna_path) + sep
+        
+        # Long Duong : Fix bug #543 (looking for Senna executable) 
+        
+        # Verifies the existence of the executable on the self._path first    
+        #senna_binary_file_1 = self.executable(self._path)
+        exeFile1 = self.executable(self._path)
+        if not path.isfile(exeFile1):
```

Comment:
ah, I see - sorry for the noise! For some reason I thought self._path is a hardcoded path.